examples/05_vol_geom_wasm_threejs/index.html

```diff
@@ -0,0 +1,330 @@
+The requestVideoFrameCallback() extension is used for synchronous video frame fetch.
+This callback is not supported on all browsers yet. https://caniuse.com/mdn-api_htmlvideoelement_requestvideoframecallback
+
+To Run:
+
+* Copy the vologram you want to play as a subfolder into this folder.
+* Set the video source and header and sequence file to point to your vologram.
+* Launch a local HTTP server from this folder.
+* Open the HTML page.
+* Hit "load vologram" to load and play the vologram.
+-->
+
+<meta charset="UTF-8">
+<meta name="author" content="Volograms Ltd.">
+<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+
+<head>
+</head>
+
+<body>
+	<input type="button" id="load_vols_files_button" value="load vologram" hidden></input>
+	<canvas id="canvas"></canvas>
+	<!-- playsinline stops mobile browsers fullscreen-ing the video (iOS 10+)-->
+	<video controls id="texture_video_el" type="video/mp4;" width="256" height="256" loop playsinline></video>
+</body>
+<script src="./vol_geom.js"></script>
+<!-- Import three.js either locally or through cdn -->
+<script src="./three.js"></script>
+<!--script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r128/three.min.js"></script-->
+
+<script>
+	var canvas;
+	var renderer;
+	var camera;
+	var vologram = new Object();   // Stores state about the current vologram.
+	vologram.header_url = "./calif/header.vols";
+	vologram.sequence_url = "./calif/sequence_0.vols";
+	vologram.video_url = "./calif/output_720.mp4";
+	vologram.fps = 30.0;           // Change this value if not using 30 frames-per-second video.
+	var init_done = false;         // If the basic shaders/objects are set up. A vologram can be downloaded & prepared after this is true.
+
+	function create_vologram_geometry() {
+		vologram.geometry = new THREE.BufferGeometry();
+		vologram.geometry.setAttribute(
+			"position", 
+			new THREE.Float32BufferAttribute(
+				new Float32Array(vologram.max_blob_size), 3, false
+			)
+		);
+		vologram.geometry.setAttribute(
+			"normal", 
+			new THREE.Float32BufferAttribute(
+				new Float32Array(vologram.max_blob_size), 3, false
+			)
+		);
+		vologram.geometry.setAttribute(
+			"uv",
+			new THREE.Float32BufferAttribute(
+				new Float32Array(vologram.max_blob_size), 2, false
+			)
+		);
+		vologram.geometry.setIndex(
+			new THREE.Uint16BufferAttribute(
+				new Uint16Array(vologram.max_blob_size), 1, false
+			)
+		);
+	}
+
+	function create_vologram_texture() {
+		var texture_video_el = document.getElementById("texture_video_el");
+		vologram.texture = new THREE.VideoTexture( texture_video_el );
+	}
+
+	function create_vologram_mesh() {
+		vologram.material = new THREE.MeshBasicMaterial({
+			color: 0xffffff,
+			map: vologram.texture,
+		});
+
+		vologram.mesh = new THREE.Mesh(vologram.geometry, vologram.material);
+		vologram.mesh.position.set( 0, 0, 0 )
+		// The vologram is mirrored
+		vologram.mesh.scale.set( -1, 1, 1 )
```

Comment:
might need to do this in the mesh update function rather, since it seems to get wiped out after each update.

the rotation only wants to be applied once though so goes here.

Reply:
That seems strange, I'll try setting the scale of the geometry scale rather than the mesh and see if that's permanent